examples/nat_topo.py

```diff
@@ -230,11 +237,15 @@ def sendInt(net, name, **args):
     # Set up the parser
     parser = argparse.ArgumentParser(description="Test the Pax NAT implementation.")
     parser.add_argument("action", choices=["run", "test"], nargs="?", default="run")
-    parser.add_argument("--no-X", help="don't launch additional windows'", action="store_false", dest="X_windows")
+    parser.add_argument("--no-X", help="don't launch additional windows", action="store_false", dest="X_windows")
+    parser.add_argument("--hold-open", help="leave xterm windows open", action="store_true", dest="hold_open")
+    parser.add_argument("--cli-first", help="provide cli access before running the tests", action="store_true", dest="cli_first")
```

Comment:
Great, thanks.

On 2016-07-14 11:18, TMVector wrote:
> In examples/nat_topo.py [1]:
> 
>> @@ -230,11 +237,15 @@ def sendInt(net, name, **args):
>> # Set up the parser
>> parser = argparse.ArgumentParser(description="Test the Pax NAT
> implementation.")
>> parser.add_argument("action", choices=["run", "test"], nargs="?",
> default="run")
>> - parser.add_argument("--no-X", help="don't launch additional
> windows'", action="store_false", dest="X_windows")
>> + parser.add_argument("--no-X", help="don't launch additional
> windows", action="store_false", dest="X_windows")
>> + parser.add_argument("--hold-open", help="leave xterm windows
> open", action="store_true", dest="hold_open")
>> + parser.add_argument("--cli-first", help="provide cli access before
> running the tests", action="store_true", dest="cli_first")
> 
> The user gets access to the Mininet CLI after the network is setup,
> but before Pax is started. They can finish the CLI with ^D and then
> the tests will start. I will clarify the help text.
> 
> --
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub [2], or mute the
> thread [3].
> 
> 	*
> 
> Links:
> ------
> [1] https://github.com/niksu/pax/pull/9#discussion_r70781977
> [2]
> https://github.com/niksu/pax/pull/9/files/6f58250655695a856c9c7e6c34ca2bb0edb2566b..d324c1fe3f83ce25053e4bc3fbfe0f249889a8de#r70781977
> [3]
> https://github.com/notifications/unsubscribe/ABDTThRj7ZsCvEOMkkli2LUW6DYncJSZks5qVg0MgaJpZM4JLXUX

-- 
http://www.cl.cam.ac.uk/~ns441/